test/tests/feedTest.js

```diff
@@ -200,7 +200,21 @@ describe("Zotero.Feed", function() {
 	});
 	
 	describe("#storeSyncedSettings", function() {
-		it("should store updated settings for the feed", function* () {
+		it("should store settings for feed with compact format", function* () {
+			let url = 'http://' + Zotero.Utilities.randomString().toLowerCase() + '.com/feed.rss';
+			let settings = [Zotero.Utilities.randomString(), 1, 1];
+			let feed = yield createFeed({
+				url, 
+				name: settings[0], 
+				cleanupAfter: settings[1], 
+				refreshInterval: settings[2]
+			});
+			
+			let syncedFeeds = Zotero.SyncedSettings.get(Zotero.Libraries.userLibraryID, 'feeds');
+			assert.deepEqual(syncedFeeds[url], settings);
+		});	
+		
+		it("should not store settings for feed with the old format", function* () {
```

Comment:
No need for this test, and can just call the other one "should store settings for the feed".